### ISSUE TITLE
Warning message

### ISSUE THREAD

**Issue by djzort** [TvdW/perl-DBD-Cassandra#18]
Duplicated from: https://rt.cpan.org/Public/Bug/Display.html?id=121985

Hello

I am getting this warning all the time from Cassandra::Client (via DBD::Cassandra):

```
Use of uninitialized value $^GLOBAL_PHASE in string eq at /usr/local/share/perl5/Cassandra/Client.pm line 396.
$ perl -v
This is perl, v5.10.1 (*) built for i386-linux-thread-multi

```
I would suggest that this code:

```
sub DESTROY {
    local $@;
    return if ${^GLOBAL_PHASE} eq 'DESTRUCT';

    my $self= shift;
    if ($self->{connected}) {
        $self->shutdown;
    }
}
```

Needs a slight change to:

```
sub DESTROY {
    local $@;
    return if (${^GLOBAL_PHASE} and ${^GLOBAL_PHASE} eq 'DESTRUCT');

    my $self= shift;
    if ($self->{connected}) {
        $self->shutdown;
    }
}
```

But, as they say in recipe books: add salt to taste.

**Comment by TvdW** [comment]
Hi Dean,

Thanks for the report! I have good news for you, this is fixed in the recently released Cassandra::Client 0.11 (2 days ago) after your bug report on RT -- forgot to close it there :)

Tom

**Comment by djzort** [comment]
cheers. thanks, i will cpanm the new version
